js/src/forum/components/SuspendUserModal.js

```diff
@@ -79,12 +55,79 @@ export default class SuspendUserModal extends Modal {
     );
   }
 
+  formItems() {
+    const items = new ItemList();
+
+    items.add(
+      'not-suspended',
+      <label className="checkbox">
+        <input type="radio" name="status" checked={!this.status()} value="" onclick={withAttr('value', this.status)} />
+        {app.translator.trans('flarum-suspend.forum.suspend_user.not_suspended_label')}
+      </label>,
+      100
+    );
+
+    items.add(
+      'indefinitely',
+      <label className="checkbox">
+        <input type="radio" name="status" checked={this.status() === 'indefinitely'} value='indefinitely' onclick={withAttr('value', this.status)} />
+        {app.translator.trans('flarum-suspend.forum.suspend_user.indefinitely_label')}
+      </label>,
+      90
+    );
+
+    items.add(
+      'time-suspension',
+      <label className="checkbox SuspendUserModal-days">
+        <input type="radio" name="status" checked={this.status() === 'limited'} value='limited' onclick={e => {
+          this.status(e.target.value);
+          m.redraw.sync();
+          this.$('.SuspendUserModal-days-input input').select();
+          e.redraw = false;
+        }} />
+        {app.translator.trans('flarum-suspend.forum.suspend_user.limited_time_label')}
+        {this.status() === 'limited' ? (
+          <div className="SuspendUserModal-days-input">
+            <input type="number"
+              min="0"
+              value={this.daysRemaining()}
+              oninput={withAttr('value', this.daysRemaining)}
+              className="FormControl" />
+            {app.translator.trans('flarum-suspend.forum.suspend_user.limited_time_days_text')}
+          </div>
+        ) : ''}
```

Comment:
A little nicer than a ternary

```suggestion
        {this.status() === 'limited' && (
          <div className="SuspendUserModal-days-input">
            <input type="number"
              min="0"
              value={this.daysRemaining()}
              oninput={withAttr('value', this.daysRemaining)}
              className="FormControl" />
            {app.translator.trans('flarum-suspend.forum.suspend_user.limited_time_days_text')}
          </div>
        )}
```